config-connector/solutions/sql/kpt/postgres-ha/sqlinstance.yaml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: postgres-ha-solution # {"$ref":"#/definitions/io.k8s.cli.setters.instance-name"}
+spec:
+  databaseVersion: POSTGRES_9_6
+  region: us-central1 # {"$ref":"#/definitions/io.k8s.cli.setters.region"}
+  settings:
+    tier: db-custom-1-3840
+    availabilityType: REGIONAL
+    ipConfiguration:
+      ipv4Enabled: true
+      requireSsl: true
+      authorizedNetworks:
+      - name: mysql-ha-solution-cidr
+        value: 192.10.10.10/32 # {"$ref":"#/definitions/io.k8s.cli.setters.external-ip-range"}
+    locationPreference:
+      zone: us-central1-c # {"$ref":"#/definitions/io.k8s.cli.setters.zone"}
+---
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLInstance
+metadata:
+  name: postgres-ha-solution-backup-1 # {"$ref":"#/definitions/io.k8s.cli.substitutions.backup-instance-1-name"}
+spec:
+  databaseVersion: POSTGRES_9_6
+  region: us-central1 # {"$ref":"#/definitions/io.k8s.cli.setters.region"}
+  settings:
+    tier: db-custom-1-3840
+    availabilityType: ZONAL
+    ipConfiguration:
+      ipv4Enabled: true
+      requireSsl: false
+      authorizedNetworks:
+      - name: mysql-ha-solution-cidr
+        value: 192.10.10.10/32 # {"$ref":"#/definitions/io.k8s.cli.setters.external-ip-range"}
+    locationPreference:
+      zone: us-central1-a # {"$ref":"#/definitions/io.k8s.cli.setters.zone-backup-1"}
+---
```

Comment:
For all 3 replica instance please insure:
1. they are called -replicas, not backup
2. masterInstanceRef set to master database
3. activationPolicy:"ALWAYS", crashSafeReplication:true, diskAutoresize:true, diskSize:10, diskType:"PD_HDD", pricingPlan: "PER_USE", replicationType: "SYNCHRONOUS", tier: "db-custom-2-13312"
For databaseFlags: name: "autovacuum", value: "off"
for maintenanceWindow: day:"7", hour: "22", updateTrack: "stable"

Reply:
Changes made